sdks/python/apache_beam/io/gcp/bigquery_file_loads.py

```diff
@@ -181,13 +181,22 @@ def display_data(self):
         'coder': self.coder.__class__.__name__
     }
 
+  def _get_hashable_destination(self, destination):
+    if isinstance(destination, bigquery_api.TableReference):
+      return '%s:%s.%s' % (
+          destination.projectId, destination.datasetId, destination.tableId)
+    else:
+      return destination
+
   def start_bundle(self):
     self._destination_to_file_writer = {}
 
   def process(self, element, file_prefix):
```

Comment:
Is it possible to add a docstring for `process` to explain what we expect as inputs? Sounds like for `element` we expect a tuple of objects with particular input types. Could we clarify that?